Sources/SwiftParser/Specifiers.swift

```diff
@@ -415,7 +415,120 @@ extension RawAccessorEffectSpecifiersSyntax: RawEffectSpecifiersTrait {
       }
     }
   }
+}
+
+extension RawDeinitEffectSpecifiersSyntax {
+  enum MisspelledAsyncSpecifiers: TokenSpecSet {
+    case await
+    case reasync
+
+    init?(lexeme: Lexer.Lexeme) {
+      switch PrepareForKeywordMatch(lexeme) {
+      case TokenSpec(.await, allowAtStartOfLine: false): self = .await
+      case TokenSpec(.reasync): self = .reasync
+      default: return nil
+      }
+    }
+
+    var spec: TokenSpec {
+      switch self {
+      case .await: return TokenSpec(.await, allowAtStartOfLine: false)
+      case .reasync: return .keyword(.reasync)
+      }
+    }
+  }
+  
+  enum MisspelledThrowsTokenKinds: TokenSpecSet {
+    case `try`
+    case `throw`
+    case `rethrows`
+    case `throws`
```

Comment:
Could you order them alphabetically? It just makes it easier to read IMO. Same in the switches below

Reply:
Done